index.html

```diff
@@ -337,6 +356,17 @@ <h2 class="section-heading animated" data-animation="bounceInUp">Team</h2>
                                        <span>Member, Embedded Team</span>
                                     </div>
                                  </div>
+                              </article>
+
```

Comment:
Can I edit it from here or do I have to send a new PR??


On Sun, Mar 13, 2016 at 5:12 PM, Abhinav Jain <notifications@github.com>
wrote:

> In index.html
> <https://github.com/Swarm-IITKgp/Swarm-IITKgp.github.io/pull/37#discussion_r55935863>
> :
>
> > @@ -337,6 +356,17 @@ <h2 class="section-heading animated" data-animation="bounceInUp">Team</h2>
> >                                         <span>Member, Embedded Team</span>
> >                                      </div>
> >                                   </div>
> > +                              </article>
> > +
>
> Did you add Shivang's information also along with your commit? He already
> did that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Swarm-IITKgp/Swarm-IITKgp.github.io/pull/37/files#r55935863>
> .
>